mtp_noms_ops/apps/user_admin/views.py

```diff
@@ -0,0 +1,17 @@
+from django.utils.translation import gettext_lazy as _
+from mtp_common.user_admin.views import UserCreationView as UserCreationViewBase
+
+from .forms import UserUpdateForm
+
+
+
```

Comment:
One too many blank lines

Reply:
Darn it, I moved things around a little and forgot a newline 😛

Reply:
https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/533/commits/5fdfda9d95b9fc23d6f8cc673fa7d8ef7de51cbb should hopefully sort this out.